Projects/Application/Sources/Flow/MainFlow/FeedScreen/FeedViewController.swift

```diff
@@ -28,5 +30,21 @@ final class FeedViewController: BaseViewController<FeedViewReactor, FeedView> {
                 cell.reactor = reactor
             }
             .disposed(by: disposeBag)
+
+        currentView.tableView.rx.contentOffset
+            .filter { [weak self] offset in
+                guard let `self` = self else { return false }
+                guard self.currentView.tableView.frame.height > 0 else { return false }
+                return offset.y + self.currentView.tableView.frame.height
+                    >= self.currentView.tableView.contentSize.height - 100
+            }
+            .map { _ in Reactor.Action.loadNextPage }
+            .bind(to: reactor.action)
+            .disposed(by: disposeBag)
+    }
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        guard let reactor = reactor?.currentState.postReactors[indexPath.row] else { return 40.0 }
```

Comment:
Same, what does `40.0` represent?

Reply:
Nothing. I think it will never return 40.0

Reply:
Then isn't it better to explicitly fail there (e.g. `assert(false)`) instead of continuing working with an invalid state?

Reply:
Fixed